### DISCUSSION TITLE
rxode2: what is new

### DISCUSSION THREAD

**Discussion by dzianismr** [nlmixr2/rxode2#211] [Q&A] [answered]
Hi Folks,

Great to see rxode2 released. Is there a summary of the key differences between rxode2 and the latest release of rxode?

**Comment by billdenney** [comment, marked as answer]
Thanks for the interest!

The list of changes is in https://github.com/nlmixr2/rxode2/blob/main/NEWS.md#rxode2-206

Most prior models will work as-is, and the list in the news file will give you the details of what has changed.
